File: codewar/train/array_diff.py
# https://www.codewars.com/kata/523f5d21c841566fde000009/train/python

def array_diff(a: list, b: list) -> list:
    res = []
    for item in a:
        if item not in b:
            res.append(item)
    
    # return res
    return [item for item in a if item not in b]



a = [1,2,2,2,3]
b = [2]
print(array_diff(a,b))

File: codewar/6-kyu/array_diff.php
<?php
// https://www.codewars.com/kata/523f5d21c841566fde000009/train/php

$a = [1, 2, 3];
$b = [1];

print_r(array_values(array_diff($a, $b)));
print_r(array_diff($a, $b));